charts/stable/gitea/questions.yaml

```diff
@@ -18,9 +18,11 @@ questions:
       attrs:
         - variable: username
           label: "username"
+          description: Set username to any user other than `admin`.
           schema:
             type: string
             required: true
+            valid_chars: ^((?!admin)[^<>])*$
```

Comment:
This also blocks any word that contains the `admin`. eg `superadmin`, `adminismyname` mradminboss`

Also what's the reasoning for blocking `<>`?

Reply:
This was copied hastely. But users have been having issues running gitea with the user `admin`.

Reply:
Agreed. I'm not much for regex though.  I was trying to use regex101 😅